samples/java-webmvc/generated-docs/index.html

```diff
@@ -1146,46 +1148,6 @@ <h4 id="_request_fields_3"><a class="anchor" href="#_request_fields_3"></a><a cl
 </colgroup>
 <thead>
 <tr>
-<th class="tableblock halign-left valign-top">Parameter</th>
```

Comment:
This section was a duplicate and this was fixed in https://github.com/ScaCap/spring-auto-restdocs/pull/393